sharding/common_test.go

```diff
@@ -137,7 +138,12 @@ func getExpandedEligibleList(num int) []uint32 {
 	sliceToRet := make([]uint32, 0)
 
 	for i := 1; i <= num; i++ {
-		randRat := rand.Intn(5) + 8
+		randBigInt, _ := rand.Int(rand.Reader, big.NewInt(5))
+		randRat := int(randBigInt.Uint64()) + 8
+		pubkey := make([]byte, 32)
```

Comment:
where is this pubKey being used?

Reply:
you right. address and public key not used anymore. was removed.